packages/react/src/components/ComposedModal/ComposedModal.js

```diff
@@ -244,7 +244,7 @@ export default class ComposedModal extends Component {
           className={`${prefix}--visually-hidden`}>
           Focus sentinel
         </span>
-        <div ref={this.innerModal} className={containerClass} tabIndex={-1}>
+        <div ref={this.innerModal} className={containerClass} role="dialog">
```

Comment:
Thank you for jumping in @kennylam! Is the removal of `tabIndex={-1}` intentional?

Reply:
Hi @asudoh, yes otherwise the element would not be tab focusable and things like screen readers would miss them, no?

Reply:
Uh, to make it keyboard-focusable it requires `tabIndex={0}`. (`tabIndex={-1}` means programmatic focusable - Ref: https://blog.whatwg.org/focusing-on-focus) However, as long as there is at least one keyboard-focusable element user can focus into the modal and use screen reader cursor to read out the content AFAIK.

Having that said, I now feel like removal of `tabIndex={-1}` is fine.. I dug the code history and `tabIndex={-1}` is there from the beginning (with no explicit reason mentioned there).